portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -60,17 +65,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String text = getParameter(request, "user-comment", "");
     long timestamp = System.currentTimeMillis();
 
-    Entity taskEntity = new Entity("Comment");
-    taskEntity.setProperty("text", text);
-    taskEntity.setProperty("timestamp", timestamp);
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("text", text);
+    commentEntity.setProperty("timestamp", timestamp);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    datastore.put(taskEntity);
+    datastore.put(commentEntity);
 
     // Redirect back to the HTML page.
     response.sendRedirect("/index.html");
   }
 
+  /** Returns the number of comments the user has chosen to display */
+  private int getAmount(HttpServletRequest request) {
+    // Get the input from the form.
+    String amtOfCommentsString = request.getParameter("display");
+
+    // Convert the input to an int.
+    int amtOfComments;
+    try {
+      amtOfComments = Integer.parseInt(amtOfCommentsString);
+    } catch (NumberFormatException e) {
+      System.err.println("Could not convert to int: " + amtOfCommentsString);
```

Comment:
Please throw an exception instead of print error

Reply:
Or you can use a Logger
https://www.vogella.com/tutorials/Logging/article.html

Reply:
Will do, thank you!